test/fuzz/readMessageBinary.js

```diff
@@ -0,0 +1,22 @@
+import openpgp from '../initOpenpgp.js';
+
+const ignored = ['This message / key probably does not conform to a valid OpenPGP format'];
```

Comment:
Very minor, but I'd consider renaming `ignored` to `expected`, which as far as I understand it's the purpose it serves..? 🙂